l10n_ec_credit_note/views/res_config_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form</field>
+        <field name="model">res.config.settings</field>
+        <field
+            name="inherit_id"
+            ref="l10n_ec_account_edi.res_config_settings_l10n_ec_edi"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='l10n_ec_section']" position="inside">
+                <block title="Credit Note" id="l10n_ec_credit_note">
```

Comment:
Remove `block` element and only keep `setting` element into another `block`
![image](https://github.com/user-attachments/assets/b9c75da9-57e0-4e6b-a543-387cf162dd1c)

Reply:
![image](https://github.com/user-attachments/assets/06a2c10d-8909-4f12-87bb-ac552e73e957)